packages/client/src/flow/sendCollectionToTreasury.tx.js

```diff
@@ -8,11 +8,6 @@ export const SEND_COLLECTION_TO_TREASURY = `
 
 		prepare(signer: AuthAccount) {
 	
```

Comment:
I'm wondering if we should keep this transaction, because now that we're adding new collections through add asset flow this is too covered. Then we will be able to remove the Deposit Collection functionality from the UI which was used just for testing. What do you think?

Reply:
yes! since now we have `add_collection` which covers this functionality already, this one can be removed.

Reply:
Removed this transaction and button which reuses it